data/sql/updates/pending_db_auth/rev_1546541118941824046.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO version_db_auth (`sql_rev`) VALUES ('1546541118941824046');
+
+ALTER TABLE `account` CHANGE `email` `email` VARCHAR(255) NOT NULL DEFAULT '';
```

Comment:
missing new line(s) at the end of file(s)

Reply:
also you can put those changes in one single sql file since it's the same database

Reply:
i separate them to be more clean

Reply:
put them in the same file please, it's more logic for our case (and with the "insert into revision" )